motoman_driver/src/industrial_robot_client/joint_trajectory_action.cpp

```diff
@@ -105,6 +108,9 @@ JointTrajectoryAction::JointTrajectoryAction() :
   pub_trajectory_command_ = node_.advertise<motoman_msgs::DynamicJointTrajectory>(
                               "joint_path_command", 1);
 
+  sub_trajectory_state_all_  = this->node_.subscribe<control_msgs::FollowJointTrajectoryFeedback>("/feedback_states", 1,
```

Comment:
Is this subscriber different than the one [here](https://github.com/ros-industrial/motoman/pull/135/files#diff-dbbdafd73ea4190550a2bc504ba53f24R87)